function-images/tests/chained-function-serving/go.mod

```diff
@@ -3,7 +3,7 @@ module tests/chained-functions-serving
 go 1.16
 
 replace (
-	github.com/ease-lab/vhive/utils/tracing/go => ./utils/tracing/go
+	github.com/ease-lab/vhive/utils/tracing/go => ../../../utils/tracing/go
```

Comment:
this doesn't seem right. how is it going to work when building in a container?

Reply:
I use the whole repo as the build context and selectively copy the directories that I need.